mirdata/datasets/compmusic_carnatic_rhythm.py

```diff
@@ -0,0 +1,382 @@
+    group), and multiple instances of the same composition rendered by different artists. Each piece is uniquely identified using the MBID of the
+    recording. The pieces are stereo, 160 kbps, mp3 files sampled at 44.1 kHz.
+
+    **SAMA AND BEATS:** The primary annotations are audio synchronized time-stamps indicating the different metrical positions in the taala cycle.
+    The annotations were created using Sonic Visualizer by tapping to music and manually correcting the taps. Each annotation has a time-stamp and
+    an associated numeric label that indicates the position of the beat marker in the taala cycle. The marked positions in the taala cycle are shown
+    with numbers, along with the corresponding label used. In each case, the sama (the start of the cycle, analogous to the downbeat) are indicated
+    using the numeral 1.
+
+    **METADATA:** For each excerpt, the taala of the piece, edupu (offset of the start of the piece, relative to the sama, measured in aksharas)
+    of the composition, and the kalai (the cycle length scaling factor) are recorded. Each excerpt can be uniquely identified and located with the
+    MBID of the recording, and the relative start and end times of the excerpt within the whole recording. A separate 5 digit taala based unique ID
+    is also provided for each excerpt as a double check. The artist, release, the lead instrument, and the raaga of the piece are additional 
+    editorial metadata obtained from the release. A flag indicates if the excerpt is a full piece or only a part of a full piece. There are optional
+    comments on audio quality and annotation specifics. 
+
+    Possible uses of the dataset: Possible tasks where the dataset can be used include taala, sama and beat tracking, tempo estimation and tracking,
+    taala recognition, rhythm based segmentation of musical audio, structural segmentation, audio to score/lyrics alignment, and rhythmic pattern
+    discovery.
+
+    Dataset organization: The dataset consists of audio, annotations, an accompanying spreadsheet providing additional metadata. For a detailed
+    description of the organization, please see the README in the dataset.
+
+    Data Subset: A subset of this dataset consisting of 118 two minute excerpts of music is also available. The content in the subset is equaivalent
+    and is separately distributed for a quicker testing of algorithms and approaches.
+
+    The annotations files of this dataset are shared with the following license:
+    Creative Commons Attribution Non Commercial Share Alike 4.0 International
+
+"""
+
+import os
+import csv
+import logging
+import librosa
+import numpy as np
+
+from mirdata import annotations, core, io, jams_utils
+from smart_open import open
+
+
+try:
+    from openpyxl import load_workbook as get_xlxs
+except ImportError:
+    logging.error(
```

Comment:
it looks like this is not covered by test? is this to be compliant with what Harsh is doing on partial dependency checks? maybe we could have a global test for that